config-templates/ibm/ocs/README.md

```diff
@@ -0,0 +1,332 @@
+
+We can also scale by adding extra disks to the exisiting worker nodes and providing the disk-paths of the new disks or, if the worker nodes already have extra local disks available, we can provide the disk-paths of those disks.  
+
+To scale the OCS cluster, we have to create a new configuration with the same name for the CRD (the parameter `ocs-cluster-name`) and the rest of the parameters should remain the same as the previous configuration, but, we should update the 'osd-device-path' parameter to include the new disk path along with the existing disk path provided previously. Similarly, if the worker nodes already have extra local disks available, we can add the disk paths of those disks in the 'osd-device-path' parameter along with the existing disk path. We should also change the value of `num-of-osd` to the scaling factor required.
+
+Example :
+
+```
+$ibmcloud sat storage config create --name ocs-config2 --template-name ocs --template-version 4.6_local -p "ocs-cluster-name=testocscluster" -p "osd-device-path=/dev/scsi-3600605b00d87b43027b3bc310a64c6c9-part2,/dev/scsi-3600605b00d87b43027b3bc310a64c6c9-part3" -p "mon-device-path=/dev/scsi-3600605b00d87b43027b3bc310a64c6c9-part1" -p "num-of-osd=2" -p "worker-nodes=169.48.170.83,169.48.170.88,169.48.170.90" -p "ibm-cos-access-key=xxx" -p "ibm-cos-secret-key=yyy"
+```
+
+After this, we need to create a new assignment for this configuration :
+
+```
+$ ibmcloud sat storage assignment create --name ocs-sub2 --group test-group2 --config ocs-config2
+```
+
+### Migration (Updating OCS version) :
+
+To update the version of OCS, we have to create a new configuration with the template version set to the version of OCS we want to upgrade to. We need to have the same name for the CRD (the parameter `ocs-cluster-name`) and the rest of the parameters should remain the same as the previous configuration, but, we should change the value of `ocs-upgrade` to true.
+
+Example :
+
+```
+$ibmcloud sat storage config create --name ocs-config3 --template-name ocs --template-version 4.7_local -p "ocs-cluster-name=testocscluster" -p "osd-device-path=/dev/scsi-3600605b00d87b43027b3bc310a64c6c9-part2,/dev/scsi-3600605b00d87b43027b3bc310a64c6c9-part3" -p "mon-device-path=/dev/scsi-3600605b00d87b43027b3bc310a64c6c9-part1" -p "num-of-osd=2" -p "worker-nodes=169.48.170.83,169.48.170.88,169.48.170.90" -p "ibm-cos-access-key=xxx" -p "ibm-cos-secret-key=yyy" -p "ocs-upgrade=true"
+```
+
+After this, we need to create a new assignment for this configuration :
+
+```
+$ ibmcloud sat storage assignment create --name ocs-sub3 --group test-group2 --config ocs-config3
+```
+
+This will upgrade OCS to the new version.
+
+**Note: Please DO NOT delete any of the older configurations and assignments as it may result in DATA LOSS**
+
+## Troubleshooting
+### What to do when OCS install fails
+- Check the describe of storagecluster and cephcluster in the openshift-storage namespace
```

Comment:
Let's show an output screen from these describe commands. Also, what messages or statuses are they looking for when they run these commands?

```suggestion
1. Describe the `storagecluster` and `cephcluster` in the `openshift-storage` namespace.
```